nixos/tests/grafana/default.nix

```diff
@@ -0,0 +1,16 @@
+{ system ? builtins.currentSystem
+, config ? { }
+, pkgs ? import ../../.. { inherit system config; }
+}:
+
+{
+  basic = import ./basic.nix { inherit system pkgs; };
+  provision-datasources = import ./provision-datasources { inherit system pkgs; };
+  provision-dashboards = import ./provision-dashboards { inherit system pkgs; };
+  provision-notifiers = import ./provision-notifiers.nix { inherit system pkgs; };
+  provision-rules = import ./provision-rules { inherit system pkgs; };
+  provision-contact-points = import ./provision-contact-points { inherit system pkgs; };
+  provision-policies = import ./provision-policies { inherit system pkgs; };
+  provision-templates = import ./provision-templates { inherit system pkgs; };
+  provision-mute-timings = import ./provision-mute-timings { inherit system pkgs; };
```

Comment:
I haven't found VM tests to be very expensive, the whole Grafana test suit takes about a minute to run on my 16GB RAM i5-10600 setup. It might be much better with #195830 merged.
Truth be told, I'm just not sure about my testing skills. Should I try to refactor the tests to use only a couple VMs in a separate commit? That would be most handy.

Reply:
Okay, actually, rewriting tests led to a 30 second win. I think that's a better solution, yes. Added a separate commit for the tests, maybe I'll actually rebase the tests later, but I'd prefer not to since this is a massive pain.